config/operator.yaml

```diff
@@ -0,0 +1,28 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: tekton-operator
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      name: tekton-operator
+  template:
+    metadata:
+      labels:
+        name: tekton-operator
+    spec:
+      serviceAccountName: tekton-operator
+      containers:
+        - name: tekton-operator
+          image: github.com/tektoncd/operator/cmd/manager
+          imagePullPolicy: IfNotPresent
```

Comment:
Not sure about this one :angel: (we don't set this on pipeline, …)

Reply:
removed it.